pages/_app.tsx

```diff
@@ -30,6 +31,11 @@ export default function App({ Component, pageProps }: AppProps) {
         <meta name="twitter:creator" content="Warga" />
         <link rel="icon" href="/favicon.ico" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <script
+          async
+          src="https://js.sentry-cdn.com/0dda1c67a0744907b58c15b965348e63.min.js"
+          crossOrigin="anonymous"
+        />
```

Comment:
After trying to run `npx @sentry/wizard -i nextjs`, all the changes other than this one are applied.

I didn't see any instruction to include this script, though. Is it even necessary?

Reply:
Yes, I think, referring to https://docs.sentry.io/platforms/javascript/install/lazy-load-sentry/. But somehow it doesn't show any effect. I'll remove it later.

Reply:
I've tried another attempt on another PR #82 and indeed, it [reduced the Performance score significantly](https://app.netlify.com/sites/wargabantuwarga/deploys/60ecf2bfb5ea9a00086e08ac), even without the `script` tag. 😅 

Hence we might need to watch [this video](https://www.youtube.com/watch?v=xEikLxEcNiU) completely and see what's we're doing wrong here. 😅